samples/client-sdk/knocking/index.js

```diff
@@ -333,23 +333,21 @@ const addOwnerEvents = () => {
 };
 
 // The owner will go right into the call since they have appropriate permissions
-const createOwnerCall = async ({ name, url, token }) => {
+const createOwnerCall = ({ name, url, token }) => {
   showLoadingText('owner');
 
   // Create call object
-  callObject = await window.DailyIframe.createCallObject();
+  callObject = window.DailyIframe.createCallObject();
 
   // Add Daily event listeners (not an exhaustive list)
   // See: https://docs.daily.co/reference/daily-js/events
   addOwnerEvents();
 
   // Let owner join the meeting
-  try {
-    await callObject.join({ userName: name, url, token });
-  } catch (error) {
+  callObject.join({ userName: name, url, token }).catch((error) => {
```

Comment:
What I meant with catching there; technically doing a try-await-catch is just syntactic sugar for this, but I figure this way we can also remove the await keyword from the definition. At least as a reader seeing a lot of async-looking logic makes me pause sometimes, so I figure since the only async thing we're doing here is catching this error this might be nicer.